src/legacy/ui/public/vis/editors/default/_sidebar.scss

```diff
@@ -225,3 +219,13 @@
   margin-top: $euiSizeS;
   margin-bottom: $euiSizeS;
 }
+
+// to apply truncated string with ellipsis for agg description
+vis-editor-agg-group-wrapper .visEditorSidebar__section > .euiFlexGroup > .euiFlexItem:first-child {
```

Comment:
Using custom tag as selector isn't a good idea, especially in our scope of removing alnular

Reply:
Fixed